diff --git a/node_modules/mediabunny/.bun-tag-5e614c7faf15aa5b b/.bun-tag-5e614c7faf15aa5b
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/bundles/mediabunny.cjs b/dist/bundles/mediabunny.cjs
index 4f4afcbea1d4bf64907ea7742d34d255f3110093..991c31dbb260de26248174f188651a27c069889c 100644
--- a/dist/bundles/mediabunny.cjs
+++ b/dist/bundles/mediabunny.cjs
@@ -24459,7 +24459,7 @@ var Mediabunny = (() => {
     }
     readU24() {
       const high = this.view.getUint16(this.pos, false);
-      const low = this.view.getUint8(this.pos + 1);
+      const low = this.view.getUint8(this.pos + 2);
       this.pos += 3;
       return high * 256 + low;
     }
diff --git a/dist/bundles/mediabunny.mjs b/dist/bundles/mediabunny.mjs
index 9b5fe8c31519a34de4ff759bcb73288153aa79ac..8d15d6a56208dbb395b910889362eaebac1fbd97 100644
--- a/dist/bundles/mediabunny.mjs
+++ b/dist/bundles/mediabunny.mjs
@@ -24308,7 +24308,7 @@ var Id3V2Reader = class {
   }
   readU24() {
     const high = this.view.getUint16(this.pos, false);
-    const low = this.view.getUint8(this.pos + 1);
+    const low = this.view.getUint8(this.pos + 2);
     this.pos += 3;
     return high * 256 + low;
   }
diff --git a/dist/bundles/mediabunny.node.cjs b/dist/bundles/mediabunny.node.cjs
index f3d3f1893f5342355b2ab7cfe038f8c923dcf010..56b4a849bd7748fefc3bbdb91122c35ac3022d56 100644
--- a/dist/bundles/mediabunny.node.cjs
+++ b/dist/bundles/mediabunny.node.cjs
@@ -24462,7 +24462,7 @@ var Mediabunny = (() => {
     }
     readU24() {
       const high = this.view.getUint16(this.pos, false);
-      const low = this.view.getUint8(this.pos + 1);
+      const low = this.view.getUint8(this.pos + 2);
       this.pos += 3;
       return high * 256 + low;
     }
diff --git a/dist/modules/src/id3.js b/dist/modules/src/id3.js
index 29e4a5a4526e0da2e6c64a0dc320c4960bcb0d68..8a2cd781dd5ff599fa73b1acc60738338a9bd067 100644
--- a/dist/modules/src/id3.js
+++ b/dist/modules/src/id3.js
@@ -442,7 +442,7 @@ export class Id3V2Reader {
     }
     readU24() {
         const high = this.view.getUint16(this.pos, false);
-        const low = this.view.getUint8(this.pos + 1);
+        const low = this.view.getUint8(this.pos + 2);
         this.pos += 3;
         return high * 0x100 + low;
     }
